server/bleep/src/webserver/answer.rs

```diff
@@ -451,6 +451,8 @@ pub async fn explain(
         start_line: params.line_start,
         end_line: params.line_end,
         snippet,
+        start_byte: 0,
```

Comment:
Again, can we set the correct byte ranges here? Are they ever used?

Reply:
the only place those are used is when getting (filtering) the subset of symbols belonging to a chunk out of all symbols in a file in `tools/symbol.rs`, that is why I added the field. 

I could change the type to `Option<usize>` for the cases where we want to create a chunk, but we don't know the exact start end bytes.